core/templates/domain/opportunity/skill-opportunity.model.ts

```diff
@@ -17,9 +17,6 @@
  * skill opportunity domain object.
  */
 
```

Comment:
Update the fileoverview* (Ditto elsewhere)

Reply:
done!